src/main/resources/templates/teams.html

```diff
@@ -112,11 +140,32 @@ <h2 class="team-heading team-heading-current-team" th:text="#{team.heading.publi
 	                               <td th:text="${teamEntry.value.description}">Description</td>
 	                               <!--<td th:text="${teamEntry.value.institution}">Institution</td>-->
 	                            </tr>
+                                <!--<tr>-->
+                                    <!--<td height="700"></td>-->
+                                <!--</tr>-->
 	                        </table>
 	                    </div> <!-- panel body -->
                     </div> <!-- panel -->
             </div> <!-- row -->
+            <div class="cc-wraper">
+                <div class="cc-contactpop fixed">
+                    <h1>Do you need Help? <span>Get in touch.</span></h1>
+                    <div class="renseignement">
+                        <i class="fa fa-times"></i>
+                        <p>We will get back to you as soon as possible.</p>
+                    </div>
+                    <form>
+                        <ul>
+                            <li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)